libs/src/api/user.js

```diff
@@ -417,9 +455,12 @@ class Users extends Base {
     }
   }
 
-  async _addUserOperations (userId, metadata) {
+  async _addUserOperations (userId, metadata, exclude = []) {
```

Comment:
for this and _updateUserOperations we should create a copy of the passed in metadata. i believe deleting the excluded keys will delete the keys in the original object that calls this function because this is pass by reference, we may be missing properties here https://github.com/AudiusProject/audius-protocol/pull/1144/files#diff-a34533690240ad151f75e407a02a875ee287c63c60bbdf814955e6ecbce52b13R412

we could do this to minimize code diff
```js
async _addUserOperations (userId, newMetadata, exclude = []) {
  const metadata = {...newMetadata}
```

Reply:
great catch, fixed